jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParams.java

```diff
@@ -134,6 +153,13 @@ public DefinitelyDerefedParams(
           } else if (instr instanceof SSAAbstractInvokeInstruction
               && !((SSAAbstractInvokeInstruction) instr).isStatic()) {
             derefValueNumber = ((SSAAbstractInvokeInstruction) instr).getReceiver();
+          } else if (instr instanceof SSAAbstractInvokeInstruction
+              && ((SSAAbstractInvokeInstruction) instr).isStatic()) {
+            SSAAbstractInvokeInstruction callInst = (SSAAbstractInvokeInstruction) instr;
+            String sign = callInst.getDeclaredTarget().getSignature();
+            if (NULL_TEST_APIS.containsKey(sign)) {
```

Comment:
Nit: Maybe add a small comment here saying that all the methods we currently support are `static` methods, hence this check being only done in this case. 

We could also generalize this to work over non-`static` `NULL_TEST_APIS` methods, but I am not sure if such a thing makes sense, since I cannot imagine a useful such method that is not `static` 😅